Cargo.toml

```diff
@@ -22,9 +25,9 @@ rapier2d = ["dep:bevy_rapier2d"]
 parallel = ["dep:rayon", "avian2d/parallel", "bevy_rapier2d/parallel"]
 
 [dependencies]
-edges = "0.4.0"
+edges = "0.4"
```

Comment:
edges is at `0.5.1` now, any reservations about bumping the dependency here to `0.5`?

Reply:
edges 0.5 depends on bevy 0.15 but rapier and avian doesn't